src/backend/web/routes/opml.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const opml = require('opml-generator');
+
+const router = express.Router();
+
+const header = {
+  title: 'OPML Feeds',
+  dateCreated: new Date(2020, 2, 9),
+  ownerName: 'Jordan',
+};
+
+const outlines = [
+  {
+    text: 'txt',
+    title: 'My Open Source Experience',
+    type: 'rss',
+    xmlUrl: 'https://jrdnlxopensource.blogspot.com/feeds/posts/default/-/open-source',
+    htmlUrl: 'https://jrdnlxopensource.blogspot.com/',
+  },
+];
+
+router.get('/opml', (req, res) => {
+  // call the opml() function here, and return it on res, using the correct content-type of " text/x-opml"
+  opml(header, outlines);
+  res.setHeader('Content-type', 'text/x-opml');
```

Comment:
Right now the return of `opml` is not used anywhere. You need to send it back with the `res`. I assume, you could use [res.send](https://sailsjs.com/documentation/reference/response-res/res-send) method for that.